FortnoxAPILibrary/UrlRequestBase.cs

```diff
@@ -92,14 +104,14 @@ internal string GetUrl(string index = "")
         internal HttpWebRequest SetupRequest(string requestUriString, string method)
         {
             Error = null;
-            if (string.IsNullOrEmpty(ConnectionCredentials.AccessToken) || string.IsNullOrEmpty(ConnectionCredentials.ClientSecret))
+            if (string.IsNullOrEmpty(_accessToken) || string.IsNullOrEmpty(_clientSecret))
```

Comment:
This condition is currently only checking the custom parameters, using the default ConnectionCredentials will yield an exception whenever a request is setup now.